crates/egui/src/containers/popup.rs

```diff
@@ -226,29 +258,27 @@ impl<'a> Popup<'a> {
     /// Sets the layout to `Layout::top_down_justified(Align::Min)`.
     pub fn menu(response: &Response) -> Self {
         Self::from_response(response)
-            .open_memory(
-                if response.clicked() {
-                    Some(SetOpenCommand::Toggle)
-                } else {
-                    None
-                },
-                PopupCloseBehavior::CloseOnClick,
-            )
+            .open_memory(if response.clicked() {
+                Some(SetOpenCommand::Toggle)
+            } else {
+                None
+            })
```

Comment:
style: this is a nice function to know about

```suggestion
            .open_memory(response.clicked().then_some(SetOpenCommand::Toggle))
```